src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx

```diff
@@ -375,13 +375,9 @@ This project was loaded using the wrong project type, likely as a result of rena
     <value>Checking items to copy to the output directory:</value>
   </data>
   <data name="FUTDC_CheckingCopyItemsForProject_1" xml:space="preserve">
-    <value>Checking copy items from project '{0}'.</value>
+    <value>Checking copy items from project '{0}':</value>
```

Comment:
@drewnoakes This change is a problem for translations. This caused our translations to get reset because it is considered to be a 'new string'. So, you can [see here](https://github.com/dotnet/project-system/pull/8745) that these will all be reset because this was merged. ETA for getting updates is ~5 days so that'll be outside of the 17.5 snap.

Best solution I can do for now is block the translations PR until after the snap occurs.

Reply:
Thanks for catching that. This string only exists in diagnostic logs which we only started translating recently, so I'm not too concerned, though of course it's good to hold off merging the reset until new strings are available.